azkaban-common/src/main/java/azkaban/executor/ExecutionController.java

```diff
@@ -0,0 +1,331 @@
+/*
```

Comment:
I've thought about creating a separate utility class to share the common methods between the new ExecutionController and old ExecutorManager. But since our plan is to deprecate ExecutorManager in the near future, I think this temporary duplication is ok for now.